packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -283,6 +283,11 @@ abstract final class PointerEventConverter {
                 position: position,
                 scrollDelta: scrollDelta,
                 embedderId: datum.embedderId,
+                acknowledge: ({bool? preventDefault}) {
+                  packet.acknowledge(
```

Comment:
This method is defined in the [engine PR](https://github.com/flutter/engine/pull/51566).